### PR TITLE
seed db

### PR DESCRIPTION
**Pull request by VALE46GP**
Hey guys.. please review these changes for me tonight.

### CODE REVIEW

File: data/dataToSeed.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    type: 'ENTIRE HOUSE',
+    beds: '3 BEDS',
+    title: 'Havalina High Desert Homestead',
+    price: 115,
+    ratings: 338,
+    average_rating: 5,
+  },
+];
+
+exports.modules = listings;
```
Comment (jcelder) [inline data/dataToSeed.js]: @VALE46GP Shouldn't this be `module.exports`?

File: data/dataToSeed.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
+
```
Comment (jcelder) [inline data/dataToSeed.js]: @VALE46GP Are you using this file at all?

File: server/index.js [hunk 3]
```diff
@@ -2,9 +2,14 @@ const express = require('express');
 
 const app = express();
 const port = 3003;
+const db = require('../data/index.js');
 // const path = require('path');
 // const bodyParser = require('body-parser');
 
 app.use('/', express.static('public'));
 
+app.get('/listings', (req, res) => {
```
Comment (jcelder) [inline server/index.js]: @VALE46GP This endpoint does not match any of your listed endpoints in your [App Service Plan](https://docs.google.com/document/d/19M5W_dSp8zcd9T9t4eLTKkGgW8KGT3Iwcu6RKzFmRxI/). Is this a different endpoint from the one listed?

File: server/index.js [hunk 4]
```diff
@@ -2,9 +2,14 @@ const express = require('express');
...
+  db.getAll();
```
Comment (jcelder) [inline server/index.js]: @VALE46GP getAll doesn't appear to be a method on the object required from (../data/index.js). Is this just a reference to a yet to be implemented method?

File: server/index.js [hunk 5]
```diff
@@ -2,9 +2,14 @@ const express = require('express');
 
 const app = express();
 const port = 3003;
+const db = require('../data/index.js');
```
Comment (jcelder) [inline server/index.js]: @VALE46GP There doesn't appear to be anything exported from this file. Additionally, is there a better name you could use to describe the object that should be exported from `../data/index.js`? IE: Are you directly interacting with the database or are you interacting with a model?

**Review by jcelder — CHANGES_REQUESTED** [review]

File: data/dataToSeed.js (hunk 2 above)
Comment (VALE46GP) [reply data/dataToSeed.js]: actually no.. i'll remove it

File: server/index.js (hunk 4 above)
Comment (VALE46GP) [reply server/index.js]: The object is used for interacting with the db but you're right; it lacked an export statement.

**Review by jcelder — APPROVED** [review]

**Pull request merged**